source/assignments.rst

```diff
@@ -2,3 +2,16 @@
 
 Assignments
 ===========
+
+
+.. _readings:
+
+Readings
+========
```

Comment:
Do we want to make this in the same page? If so should it be the same header level? I also go this when building it:

```
/home/lance/git/github/cs312/source/index.rst:45: WARNING: toctree contains reference to nonexisting document u'readings'
```